torch/_torch_docs.py

```diff
@@ -4731,16 +4731,27 @@ def merge_dicts(*dicts):
            r"""
 t(input) -> Tensor
 
-Expects :attr:`input` to be a matrix (2-D tensor) and transposes dimensions 0
+Expects :attr:`input` to be a 0-D or 1-D or 2-D tensor and transposes dimensions 0
 and 1.
 
-Can be seen as a short-hand function for ``transpose(input, 0, 1)``.
+0-D and 1-D tensors are returned as it is and
+2-D tensor can be seen as a short-hand function for ``transpose(input, 0, 1)``.
 
 Args:
     input (Tensor): the input tensor
 
 Example::
 
+    >>> x = torch.randn(())
+    >>> x
+    tensor(0.1995)
+    >>> torch.t(x)
+    tensor(0.1995)
+    >>> x = torch.randn((3))
```

Comment:
`torch.randn((3))` is the same as `torch.randn(3)`.  If you want to pass a tuple that is the same as `shape` (it will still have the same effect), you want `torch.randn((3,))

Reply:
done